docs/reset-security-verification-for-wog-account.md

```diff
@@ -1,4 +1,4 @@
-# Reset security verification for WOG account
+# Reset MFA for WOG account
 
 This article is applicable only for public officers. Public officers may have to reset security verification for their WOG account in the following cases:
```

Comment:
```suggestion
This article is applicable only for public officers. Public officers may have to reset security verification (SG Govt M365 MFA) for their WOG account in the following cases:
```

Reply:
it is clear to me that security verification is reset for SG Govt M365 profile of the user's WOG account.  if the device used for MFA is lost, I understand a new device will be used and sometimes there may be a change in the phone number.

So does it mean that security verification is reset for SG Govt M365 profile of the WOG account and for the new mobile number? 

"etc" - what are the other things for which the security verification is reset?

Reply:
ah.. ignore this comment. this is the additional security verification page. 

there's a security verification page that captures other info like mobile number and email.. those are only applicable to vendor accounts. details captured here are used by microsoft to allow self password reset. techpass don't have access to it. and self password reset via browser is not permitted for public officers.

my mistake. sorry for confusion

Reply:
no problem :-) Thanks!